_header

```diff
@@ -0,0 +1,10 @@
+Cache-Control: no-cache, no-store, max-age=0, must-revalidate
+Pragma: no-cache
+Expires: 0
+X-Content-Type-Options: nosniff
+X-Frame-Options: DENY
+X-XSS-Protection: 1; mode=block
+referrer-policy: no-referrer, strict-origin-when-cross-origin
+Strict-Transport-Security: max-age=31536000 ; includeSubDomains
+content-security-policy: default-src 'none'; script-src 'self' 'unsafe-inline' ajax.googleapis.com cdn.syndication.twimg.com; connect-src 'self'; img-src 'self' data: pbs.twimg.com *.travis-ci.org travis-ci.org; style-src 'self' 'unsafe-inline' fonts.googleapis.com; font-src 'self' data: fonts.gstatic.com; object-src 'none'; frame-ancestors 'none';report-uri https://unic.report-uri.com/r/d/csp/enforce; report-to csp
```

Comment:
Where you able to verify that this content-security-policy is complete?

Reply:
I tested the policy according to https://projects.unic.com/x/G9Px. If it's not complete we'll get reports and can add any missing items.

Reply:
Perfect, thank you!